src/main/java/cs2103/v15_1j/jolt/command/ChangeCommand.java

```diff
@@ -103,13 +105,25 @@ public UIFeedback execute(ControllerStates conStates) {
             if (newStartDate != null) {
             	if (actual instanceof DeadlineTask) {
             		deadlineTask.setDate(newStartDate);
+            	} else if (actual instanceof FloatingTask) {
+            		conStates.masterList.remove(actual);
+            		if (newStartTime != null) {
```

Comment:
also, what if the user type "reschedule f1 to 7pm"? I think you should convert it to a deadline task as well. so you should add something similar to the `if (newStartTime != null)` part below